less/tabzilla.less

```diff
@@ -0,0 +1,38 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "lib/lib.less";
```

Comment:
That was my thoughts as well but, seeing the entire `lib.less` is not actually merged into tabzilla.css I felt it would be ok. With that said, I am completely open to trimming it down.